### PR TITLE
MileStone 1 | List Structure

### PR DESCRIPTION
**Pull request by alexander16108**
## Project requirements
- I have set up a new project with webpack that is based on the webpack exercise i completed.

- I created an index.html file.

- I  create an empty To-Do List placeholder (`<div>` or `<ul>` element). The index.html file is set as s template using the HTML Webpack Plugin.

- Create an index.js file and set an array of some simple to do tasks (array of objects). Each task object should contain three 
  #### keys:
> description [string].
> completed [bool].
> index: [number].
- Write a function to iterate over the tasks array and populate an HTML list item element for each task.

- On page load render the dynamically created list of tasks in the dedicated placeholder. The list should appear in order of the index values for each task.

- Create a style.css and set rules for the To Do List.

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,35 @@
+<!DOCTYPE html>
```
Comment (carlos-ssh) [inline dist/index.html:1]: - Kindly add  a **README** file description about you and your project.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
Hi @alexander16108 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md [hunk 2]
```diff
@@ -1 +1,55 @@
...
+#### Acknowledgments
+- Microverse.org for assigning us this project
+
+- The Amazing staff for working round the Clock to make our journey free of bumps.
+
+- The amazing code reviewers for making us improve every day :thumbsup:
+
+### Licence 
+
+This Project is MIT licensed
```
Comment (carlos-ssh) [inline README.md:55]: [OPTIONAL]
- Provide a link to your License.

**Review by carlos-ssh — APPROVED** [review]
Hi @alexander16108 , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: README.md (hunk 2 above)
Comment (alexander16108) [reply README.md:55]: ok thanks i will

**Pull request merged**